src/server.c

```diff
@@ -3161,13 +3183,8 @@ void call(client *c, int flags) {
 
     server.in_nested_call--;
 
-    /* Update failed command calls if required.
-     * We leverage a static variable (prev_err_count) to retain
-     * the counter across nested function calls and avoid logging
-     * the same error twice. */
```

Comment:
we can maybe keep that comment.
also since you did a nice trick of using the same function to count the failures and reset the error counter, it can remain a static variable of that function rather than a global.

Reply:
Fixed